tests/services/inventory/test_data/additional_cai_resources.dump

```diff
@@ -2,7 +2,9 @@
 # enabled. These resources are merged into mock_cai_resources.dump by the
 # update_cai_dumps.py script. Any lines that start with a '#' are ignored.
 #
-# Bigquery Dataset
+# Bigquery Dataset with project id
+{"name":"//bigquery.googleapis.com/projects/bigquerytestdefault/datasets/bq_test_ds","asset_type":"google.cloud.bigquery.Dataset", "resource":{"version":"v2","discovery_document_uri":"https://bigquery.googleapis.com/$discovery/rest","discovery_name":"Dataset","parent":"//cloudresourcemanager.googleapis.com/projects/1042","data":{"creationTime":"1540220016171","datasetReference":{"datasetId":"bq_test_ds","projectId":"project2"},"location":"us","id":"project2:bq_test_ds","kind":"bigquery#dataset","lastModifiedTime":"1540220016171"}}}
```

Comment:
bigquerytestdefault needs to be project2 to match the other data that is used by the crawler.
bq_test_ds should be a different name, like bq_test_ds2

Reply:
Updated to project3